### ISSUE TITLE
MKLPardisoKKTSolver not correctly exported in namespace

### ISSUE THREAD

**Issue by Sage0614** [oxfordcontrol/COSMO.jl#159]
I am trying to use MKLPardisoKKTSolver in JuMP

`model = JuMP.Model(optimizer_with_attributes(COSMO.Optimizer, "kkt_solver" => with_options(PardisoDirectKKTSolver, msg_level_on = true));`

and getting the error message UndefVarError: MKLPardisoKKTSolver not defined

I think the problem is with this file:
https://github.com/oxfordcontrol/COSMO.jl/blob/8689f7611d0e8508b5e75d00b27cf33a39c32a1b/src/linear_solver/kktsolver_pardiso.jl#L3

MKLPardisoKKTSolver rather than MKLPardisoSolver should be exported to namespace.

Thanks,

**Comment by migarstka** [comment]
Hi @Sage0614,

You are right, I fixed it in this commit 8d0f108: (Haven't released it  yet):
If you checkout the commit or the master branch, the following script should work:


```julia
using Pardiso, COSMO, JuMP, LinearAlgebra, SparseArrays, Test, Random

rng = Random.MersenneTwister(12345);
## create a random test matrix C
n = 8;
C = -1 .+ rand(rng, n, n) .* 2;
c = vec(C);

# Define problem in `JuMP`:
q = -vec(C);
r = 0.5 * vec(C)' * vec(C);
m = JuMP.Model(optimizer_with_attributes(COSMO.Optimizer, "verbose" => true, "kkt_solver" => MKLPardisoKKTSolver));
@variable(m, X[1:n, 1:n], PSD);
x = vec(X);
@objective(m, Min, 0.5 * x' * x  + q' * x + r);
for i = 1:n
  @constraint(m, X[i, i] == 1.);
end

# Solve the `JuMP` model with `COSMO` and query the solution `X_sol`:
status = JuMP.optimize!(m);
obj_val = JuMP.objective_value(m);
X_sol = JuMP.value.(X);

```

**Comment by Sage0614** [comment]
> Hi @Sage0614,
> 
> You are right, I fixed it in this commit [8d0f108](https://github.com/oxfordcontrol/COSMO.jl/commit/8d0f1083e7bc2f855f7fd852848f0f4f860d2611): (will be part of new releave `v0.8.6`): If you checkout the commit or the master branch, the following script should work:
> 
> (Notice you need to load Pardiso `using Pardiso` independently.
> 
> ```julia
> using Pardiso, COSMO, JuMP, LinearAlgebra, SparseArrays, Test, Random
> 
> rng = Random.MersenneTwister(12345);
> ## create a random test matrix C
> n = 8;
> C = -1 .+ rand(rng, n, n) .* 2;
> c = vec(C);
> 
> # Define problem in `JuMP`:
> q = -vec(C);
> r = 0.5 * vec(C)' * vec(C);
> m = JuMP.Model(optimizer_with_attributes(COSMO.Optimizer, "verbose" => true, "kkt_solver" => MKLPardisoKKTSolver));
> @variable(m, X[1:n, 1:n], PSD);
> x = vec(X);
> @objective(m, Min, 0.5 * x' * x  + q' * x + r);
> for i = 1:n
>   @constraint(m, X[i, i] == 1.);
> end
> 
> # Solve the `JuMP` model with `COSMO` and query the solution `X_sol`:
> status = JuMP.optimize!(m);
> obj_val = JuMP.objective_value(m);
> X_sol = JuMP.value.(X);
> ```

That works fine for me, think you can close the issue.
